Snippets/Core/Core_7.7/Core_7.7.csproj

```
@@ -6,7 +6,7 @@
     <LangVersion>7.3</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="7.7.2" />
+    <PackageReference Include="NServiceBus" Version="7.7.0" />
```

This seems to be an exact version. Those can be set in brackets
```suggestion
    <PackageReference Include="NServiceBus" Version="[7.7.0]" />
```

See https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges

---

FYI we have done a silent release to myget using 7.8. Our goal is to release this probably mid next week

---

I rolled back the change to `7.7.2`. I think the change to how we specify exact versions should be done consistently across the board.